OpenGL/Math/Matrix4.cs

```diff
@@ -108,6 +108,73 @@ public static Matrix4 Identity
             }
         }
 
+        public float this[int column, int row]
```

Comment:
I would suggest doing this in a couple of parts.  `Vector3` and `Vector2` already support `this[int a]` for accessing individual fields.  I would suggest extending `Vector4` to also support this, which will greatly simplify the Matrix4 implementation.

You'll notice the `Vector2` and `Vector3` versions check for bounds and throw an exception.  I believe this code should do the same, checking to ensure row is in range.  The column value will then be checked by the `Vector4` code.